### PR TITLE
Clarify MachineHealthChecks for Masters

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1869210

### CODE REVIEW

**Comment by mburke5678** [comment]
@simonkrenger PTAL

**Comment by simonkrenger** [comment]
@mburke5678 LGTM

**Comment by mburke5678** [comment]
@mike-nguyen PTAL

**Comment by mburke5678** [comment]
@mike-nguyen PTAL

**Comment by mburke5678** [comment]
@jhou1 Can you please assign this PR to the appropriate member of your team for review? Thank you in advance.

**Comment by jhou1** [comment]
@sunzhaohua2 PTAL, thanks.

**Comment by sunzhaohua2** [comment]
/lgtm
thank you!

File: modules/machine-health-checks-about.adoc [hunk 1]
```diff
@@ -8,13 +8,25 @@
 
 Machine health checks automatically repair unhealthy machines in a particular machine pool.
 
-To monitor machine health, you create a resource to define the configuration for a controller. You set a condition to check for, such as staying in the `NotReady` status for 15 minutes or displaying a permanent condition in the node-problem-detector, and a label for the set of machines to monitor.
+To monitor machine health, create a `MachineHealthCheck` (CR) custom resource that includes a label for the set of machines to monitor and a condition to check, such as staying in the `NotReady` status for 15 minutes or displaying a permanent condition in the node-problem-detector.
```
Comment (ahardin-rh) [inline modules/machine-health-checks-about.adoc:11]: should this be custom resource (CR)?

File: modules/machine-health-checks-about.adoc [hunk 2]
```diff
@@ -8,13 +8,25 @@
 
 Machine health checks automatically repair unhealthy machines in a particular machine pool.
 
-To monitor machine health, you create a resource to define the configuration for a controller. You set a condition to check for, such as staying in the `NotReady` status for 15 minutes or displaying a permanent condition in the node-problem-detector, and a label for the set of machines to monitor.
+To monitor machine health, create a `MachineHealthCheck` (CR) custom resource that includes a label for the set of machines to monitor and a condition to check, such as staying in the `NotReady` status for 15 minutes or displaying a permanent condition in the node-problem-detector. 
+
+The controller that observes a `MachineHealthCheck` (CR) custom resource checks for the condition that you defined. If a machine fails the health check, the machine is automatically deleted and a new one is created to take its place. When a machine is deleted, you see a `machine deleted` event.
```
Comment (ahardin-rh) [inline modules/machine-health-checks-about.adoc:13]: If you have custom resource (CR) defined above, then you can just say `MachineHealthCheck` CR here

File: modules/machine-health-checks-about.adoc [hunk 3]
```diff
@@ -8,13 +8,25 @@
...
-You cannot apply a machine health check to a machine with the master role.
-====
+For machines with the master role, the machine health check reports the number of unhealthy nodes, but the machine is not deleted. For example:
 
-The controller that observes a `MachineHealthCheck` resource checks for the status that you defined. If a machine fails the health check, it is automatically deleted and a new one is created to take its place. When a machine is deleted, you see a `machine deleted` event. To limit disruptive impact of the machine deletion, the controller drains and deletes only one node at a time. If there are more unhealthy machines than the `maxUnhealthy` threshold allows for in the targeted pool of machines, remediation stops so that manual intervention can take place.
+[source,terminal]
+----
+$ oc get machinehealthcheck example -n openshift-machine-api
+----
+[source,terminal]
```
Comment (ahardin-rh) [inline modules/machine-health-checks-about.adoc:23]: .Example output

File: modules/machine-health-checks-about.adoc [hunk 4]
```diff
@@ -8,13 +8,25 @@
...
+----
+NAME      MAXUNHEALTHY   EXPECTEDMACHINES   CURRENTHEALTHY
+example   40%            3                  1
+----
+
+To limit the disruptive impact of machine deletions, the controller drains and deletes only one node at a time. If there are more unhealthy machines than the `maxUnhealthy` threshold allows for in the targeted pool of machines, the controller stops deleting machines and you need to manually intervene.
```
Comment (ahardin-rh) [inline modules/machine-health-checks-about.adoc:29]: s/you need to/you must

**Comment by ahardin-rh** [comment]
@mburke5678 Just some minor suggestions from me. Looks great!

**Pull request merged**